### PR TITLE
waits for requestheader-client-ca-file in extension-apiserver-authentication  before rolling out a new version

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
I didn't validate against `master` because the bundle and the tools I got were for `4.3` version which was successfully validated with https://github.com/openshift/cluster-openshift-apiserver-operator/pull/326

File: pkg/operator/workloadcontroller/workload_controller.go [hunk 1]
```diff
@@ -160,18 +162,25 @@ func (c OpenShiftAPIServerOperator) sync() error {
...
 		if apierrors.IsNotFound(err) {
 			klog.Infof("Waiting for %q configmap in %q namespace to be available", "extension-apiserver-authentication", metav1.NamespaceSystem)
 			return nil
 		}
 		if err != nil {
 			return err
 		}
+
+		if len(authConfigMap.Data["requestheader-client-ca-file"]) == 0 {
+			klog.Infof("missing requestheader-client-ca-file filed in %q configmap", "extension-apiserver-authentication")
```
Comment (sttts) [inline pkg/operator/workloadcontroller/workload_controller.go]: .V(2)

Comment (sttts) [reply pkg/operator/workloadcontroller/workload_controller.go]: Maybe better: `waiting for requestheader-client-ca-file in ... to be populated`

**Pull request merged**